OutOfSchool/OutOfSchool.BusinessLogic/Models/Workshops/WorkshopProviderViewCard.cs

```diff
@@ -22,8 +22,8 @@ public static WorkshopProviderViewCard ToProviderViewCard(this Workshop model)
         return new()
         {
             Id = model.Id,
-            ProviderTitle = model.ProviderTitle,
-            ProviderTitleEn = model.ProviderTitleEn,
+            ProviderTitle = model.Provider?.FullTitle,
```

Comment:
more details in predicate comment, i won't write it for every DTO because it is mostly critical for types that we return in lists, such as cards.

Make sure the provider is eagerly included in the request to database, so we avoid n+1 query explosion for lists